src/libraries/System.Net.Sockets/src/System/Net/Sockets/SocketPal.Unix.cs

```diff
@@ -154,8 +171,32 @@ private static unsafe int SysReceive(SafeSocketHandle socket, SocketFlags flags,
             return checked((int)received);
         }
 
+        private static unsafe int SysWrite(SafeSocketHandle socket, ReadOnlySpan<byte> buffer, ref int offset, ref int count, out Interop.Error errno)
+        {
+            int sent;
+
+            fixed (byte* b = &MemoryMarshal.GetReference(buffer))
+            {
+                sent = Interop.Sys.Write(socket, &b[offset], buffer.Length);
```

Comment:
Is it correct that this is passing buffer.Length rather than count?

Reply:
Good catch. This must be `count`.